index.ts

```diff
@@ -641,8 +644,8 @@ collect.set('isNewRepoTemplate', [
 /** Get the logged-in user’s username */
 const getUsername = (): string | undefined => document.querySelector('meta[name="user-login"]')?.getAttribute('content')!;
 
-/** Drop leading and trailing slashes */
-const getCleanPathname = (url: URL | HTMLAnchorElement | Location = location): string => url.pathname.slice(1, url.pathname.endsWith('/') ? -1 : undefined);
+/** Drop all leading and trailing slashes */
+const getCleanPathname = (url: URL | HTMLAnchorElement | Location = location): string => url.pathname.replace(/^\/+(.*?)\/*$/, '$1');
```

Comment:
This covers slashes in the middle too

```suggestion
const getCleanPathname = (url: URL | HTMLAnchorElement | Location = location): string => url.pathname.replaceAll('//', '/').slice(1, url.pathname.endsWith('/') ? -1 : undefined);
Write
```

Reply:
Sometimes the slashes in the middle do make a difference though (leading to the 403 page), however I can't find an example like I used to see now.

Reply:
> slashes in the middle do make a difference

I doubt that's intended/acceptable, URLs should not have multiple slashes

Reply:
> > slashes in the middle do make a difference
> 
> I doubt that's intended/acceptable, URLs should not have multiple slashes

In case I didn't make it clear, I was only referring to GitHub, not in a general sense. That is indeed, as you described, an _acceptable_ (or let's say, fault-tolerant) behavior. However we are not GitHub, we can't ensure that multiple slashes in the middle does not matter.

Reply:
This PR is about dropping duplicate slashes.

Drop all duplicate slashes or close the PR. Don't claim that duplicate leading/trailing slashes are not important and then argue that duplicate slashes in the middle are important. It's either both or neither.

Reply:
> Don't claim that duplicate leading/trailing slashes are not meaningful and then argue that duplicate slashes in the middle are.

I didn't describe them as "meaningful", I was just pointing out neither way covers all cases. Skipping slashes in the middle ensure the pathname before and after processing return the same page, which is better in my opinion.

I do understand your intentions though, as I said I struggle to find the page I used to notice things are different, so those edge cases are rare enough to be safely ignored.